txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+# TX-TL Liposome Using Water-in-Oil Emulsion
+
+Version 0.1.0, 8 August 2018
+
+Contributors:
+Aaron Engelhart (University of Minnesota),
+Jan Gregrowicz (Caltech),
+Joseph Heili (University of Minnesota),
+Zoila Jurado (Caltech),
+Neha Kama (Northwestern),
+Akshay Maheshwari (Stanford),
+Richard Murray (Caltech),
+Milena Popovic (Blue Marble Space),
+Kazhito Tabata (University of Tokyo),
+Paola Torre (University of Pennsylvania).
+
+## Overview and Materials
+
+This protocol describes how to create liposomes that contain TX-TL
+inside of a lipid-based container. The liposomes created by these
+protocols are between 1 and 10 um in diameter and should be usable
+with an cell-free protein expression system.
+
+The preparation of liposomes using w/o emulsions as template requires
+four steps:
+
+### **Step 0**: Deposition of a thin lipid film in a glass vial
+
+#### Materials:
+
+* Glass vials (any will do, but we use [2mL Fisherbrand Class B Clear
+  Glass Threaded vials: cat #:
+  03-339-21A](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738)
+
+* POPC in chloroform: [https://avantilipids.com/product/850457](https://avantilipids.com/product/850457) 
+
+* Lyss-Rhod-PE:
+  [https://avantilipids.com/product/810150](https://avantilipids.com/product/810150)
+
+* Cholesterol in chloroform (25 mg/ml): https://avantilipids.com/product/700000
+
+* Glass syringes of various sizes (10 uL, 50 uL, 250 uL, and 1 mL are
+  suggested ex. Hamilton gastight cat #: 14-815-238, but any will do)
+  for use with lipids and chloroform
+  [https://www.hamiltoncompany.com/products/syringes-and-needles/general-syringes/gastight-syringes](https://www.hamiltoncompany.com/products/syringes-and-needles/general-syringes/gastight-syringes)
+
+### **Step 1**: Preparation of the oil containing lipids
+
+#### Materials:
+
+* Mineral oil: [https://us.vwr.com/store/product/7422728/mineral-oil-light-white-high-purity-grade](https://us.vwr.com/store/product/7422728/mineral-oil-light-white-high-purity-grade)
+
+### **Step 2**: Self-assembly of the Liposomes by centrifugation
+
+#### Materials:
+
+* 20 uM of HPTS stock solution: ([https://www.thermofisher.com/order/catalog/product/H348](https://www.thermofisher.com/order/catalog/product/H348))
+
+* 100 mM HEPES, 200 mM glucose pH 8
+
+* 2M Sucrose Stock
```

Comment:
I think this should be removed (was struck out in Google Doc, but formatting lost in transfer).

Reply:
Yes, now it is unnecessary because in liness 63 and 65 the "final" concentration is given. Everyone will prepare these solution as he/she wish. I suggested the 2M stock solutions because when TX-TL reaction will be implemented, these are the best stock solution to use.